addons/portal/controllers/portal.py

```diff
@@ -167,6 +186,100 @@ def _prepare_portal_overview_values(self):
             }]
         }
 
+    def _is_own_partner_address(self, partner, address_id):
+        """Checks whether address_id is the id of a res.partner representing:
+            - the main address of partner (partner.id)
+            - an invoice address from partner.child_ids
+            - a delivery address from partner.child_ids
+        """
+        # if address_id is partner's main address
+        if partner.id == address_id:
+            return True
+        # if address_id partner's secondary address
+        return address_id in partner.child_ids.filtered(lambda partner: partner.type in ['invoice', 'delivery']).ids
+
+    def _render_portal_my_address(self, values):
+        response = request.render("portal.portal_my_address", values)
+        response.headers['X-Frame-Options'] = 'DENY'
```

Comment:
mmh ? What does it do ?

Reply:
this line was already there in the original code.
from mdn web doc:
If you specify DENY, not only will the browser attempt to load the page in a frame fail when loaded from other sites, attempts to do so will fail when loaded from the same site.

Reply:
Ok ^^ could be great to dig a bit see why we have that specific here, and not anywhere else. Maybe to note in a "tocheck post merge" ?

Reply:
I'm not that knowledgeable about this, but I guess it is a protection which can be useful to prevent form-data interception with frames ?
I found this commit :
https://github.com/odoo/odoo/commit/6d16915d397db3d92795a19800f697226dcbc754